dev/run-tests.sh

```diff
@@ -29,9 +31,11 @@ FWDIR="$( cd "$DIR"/.. && pwd )"
 cd "$FWDIR"
 
 if [ "$#" = 0 ]; then
-    ARGS="--nologcapture --all-modules --verbose "
+    ARGS="--verbose --showlocals --doctest-modules databricks"
 else
     ARGS="$@"
 fi
-# Runs the main program that finds and runs the docstrings.
-python databricks/koalas/testing/doctest_main.py && nosetests $ARGS --where "$FWDIR"
+
+pushd "$FWDIR" > /dev/null
+pytest $ARGS
+popd > /dev/null
```

Comment:
why do you need that?

Reply:
Maybe we can add some more commands in this script after this, for instance, coverage report. So, I came back to the original location although it might not affect anything.

(honestly, I just copied and pasted this pattern from Spark's scripts .. and it made me feel complete when something goes in and come back .. :D..)